frontend/lib/shared/success_page.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:flutter_starter/features/app/app_tabs.dart';
+import 'package:flutter_starter/l10n/app_localizations.dart';
+import 'package:flutter_starter/shared/grid.dart';
+
+class SuccessPage extends HookWidget {
+  final String text;
+  const SuccessPage({required this.text, super.key});
```

Comment:
Nice! This is a great little reusable widget.

In the future, we might want to pass in the rout that we navigate to (with what you have here as a default). No changed needed now, just something that popped in my head.